### DISCUSSION TITLE
What's the proper format for using multiple relays in the command line?

### DISCUSSION THREAD

**Discussion by nima1981** [0xtrr/nostr-tool#40] [Q&A] [answered]
I've tried this formatting:

nostr-tool -r wss://nostr.oxtr.dev,wss://blabla.dev,wss://blabla2.dev -p {PRIVATE_KEY} text-note -c "Hello World"

The message returns like it was a successful submit event, including the note id. However the message never becomes visible when I check my nostr profile. It's like they're never actually submitted successfully and I'm unable to find the note anywhere publicly. It only works when I submit to a single relay.

Is something wrong with the formatting I'm using?

**Comment by 0xtrr** [comment, marked as answer]
Hi, good question! You should be able to do it like this:
`nostr-tool -r wss://nostr.oxtr.dev -r wss://relay.snort.social`

multiple -r flags will be appended to a list in the program

**Reply by 0xtrr** [reply]
I will try to make this more obvious in the readme soon

**Reply by nima1981** [reply]
Amazig, that did it!!! :D Thanks so much!!

**Reply by nima1981** [reply]
I'm sorry, I think I spoke too soon. It seems like the method you suggested only submits the message to the relay mentioned in the very last -r argument, overriding any previous arguments submitted.

**Reply by nima1981** [reply]
Actually correction, it's all working great, it's showing up under all relays now!
